winterfell/webapp/src/js/formCtrl.js

```diff
@@ -9,6 +9,29 @@ app.controller("formCtrl", ['$scope', 'formRenderingService', function($scope, f
     });
 
     $scope.render = function() {
-        formRenderingService.render('VBA-21-0966-ARE', [{fieldName: 'veteran_first_name', fieldValue: 'jeff'}]);
-    }
+        var out = [];
+        for (var k in $scope.model) {
+            if ($scope.model.hasOwnProperty(k)) {
+                out.push({fieldName: k, fieldValue: $scope.model[k]})
+            }
+        }
+        formRenderingService.render('VBA-21-0966-ARE', out);
```

Comment:
Agree just had this in there as I am developing, planning to get it from route params eventually, or maybe have a mapping from route params to constants